Scripts/Modificar.js

```diff
@@ -0,0 +1,21 @@
+/*"use strict";*/
+const cartas = document.querySelectorAll(".card");
+var paginaseleccionada = "primeros datos";
+
+function redirigir() {
+    paginaseleccionada = this;
+    /*paginaseleccionada = this;*/
+    /*alert(paginaseleccionada.dataset.carta);*/
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456239)
```suggestion
  /*alert(paginaseleccionada.dataset.carta);*/
```